src/libponyrt/sched/scheduler.h

```diff
@@ -41,6 +46,7 @@ struct scheduler_t
 {
   // These are rarely changed.
   pony_thread_id_t tid;
+  int index;
```

Comment:
This would be best as an `uint32_t`, the same type used in the scheduler initialisation loop. Alteratively, this could use the thread ID instead if it makes sense in DTrace. I'm not an expert here so I don't know if would really appropriate.